driver/src/metrics/stablex_metrics.rs

```diff
@@ -21,31 +21,40 @@ pub struct StableXMetrics {
 impl StableXMetrics {
     pub fn new(registry: Arc<Registry>) -> Self {
         let processing_time_opts = Opts::new(
-            "processing_times",
```

Comment:
Should these be prefixed with `$NETWORK` as well so we can distinguish rinkeby from mainnet or are we not going to record prometheus metrics for rinkeby?

Reply:
I believe filtering by network/solver strategy can be done with the host label that the event emits. I think the global prefix is just so that we don't confuse the general source of the metrics.